src/lib/containers/row_renderers/utils/Icon.tsx

```diff
@@ -140,15 +143,12 @@ const Icon: React.FunctionComponent<IconProps> = ({
   // or if the type of card has a corresponding icon, e.g. 'Publication'
   const Icon = iconSet[value] || iconSet[type]
   // TODO: get rid of dataset icon class, none of the icons should be special cased
-  const datasetIconClass =
-    value === DATASET || type === DATASET ? 'SRC-datasetIcon' : ''
-  if (typeof Icon == 'function') {
-    return (
-      <span className={cssClass}>
-        <Icon />
-      </span>
-    )
-  }
-  return <img src={Icon} className={`iconImg ${datasetIconClass}`} />
+  const datasetCustomStyle =
+    value === DATASET || type === DATASET ? { height: '55px' } : {}
```

Comment:
AFAIK this datasets icon only appears in CardContainer, so I just tweaked this to make it look right.